lib/server.js

```diff
@@ -384,7 +384,8 @@ Server.prototype.start = function start(cb) {
     debug('emitting listening event');
     var addr = self._httpServer.address();
     self.emit('listening', addr);
-    self._meshApp.emit('started', self.port);
+    self._meshApp.set('url', util.format('http://127.0.0.1:%d', addr.port));
```

Comment:
The behaviour this is polyfillig only shows localhost: https://github.com/strongloop/loopback/blob/e68cd8fc87730404a78e1475b828886e3582bd6f/lib/application.js#L545